### DISCUSSION TITLE
Export CSV show SQL Script

### DISCUSSION THREAD

**Discussion by hasanyildiz** [dbeaver/dbeaver#13446] [General]
When exporting a select statement result to csv format, data is exported well.However in csv file, i cannot see which select query is executed.It would be useful when sending csv file to someone who wants to see which result belongs to which query.

In my opinion, adding an additional sheet in csv file to show select query would be useful.

**Comment by Matvey16** [comment]
@hasanyildiz Unfortunately we can't add this to CSV exports, because it will break the standard layout of csv file, because having additional sheets is not normal for csv format that is usually opened in more basic text editors like Notepad, and will negatively impact the import from it. For now we have exactly this feature for exports into XLSX format. You need to install office support extension to use it.
